lib/i18n/wallet.i18n.dart

```diff
@@ -15,9 +15,11 @@ static var _t = Translations.byLocale("en_us") +
         "Transfer amount is not valid.": "Monto a transferir no es válido",
         "Transfer amount cannot be greater than availabe balance.": "Monto a transferir no puede ser mayor al balance disponible",
         'Send': "Enviar",
+        'Receive': "Cobrar",
```

Comment:
I am not sure about context here. But Receive = Recibir

Reply:
You know Spanish?? Well that's amazing!